File: week-02/day-03/04_takes_longer.py
# When saving this quote a disk error has occured. Please fix it.
# Add "always takes longer than" between the words "It" and "you"

quote = "Hofstadter's Law: It you expect, even when you take into account Hofstadter's Law."
new_words = "always takes longer than"

splitted_quote = quote.split(" ")
index_quote = splitted_quote.index("you")
splitted_quote.insert(index_quote, new_words) 
solution = " ".join(splitted_quote)

print(solution)

File: week-07/day-03/01/04 takes-longer.js
'use strict';
// When saving this quote a disk error has occured. Please fix it.
// Add "always takes longer than" to between the words "It" and "you"

var quote = "Hofstadter's Law: It you expect, even when you take into account Hofstadter's Law."

quote = quote.split('It you').join('It always takes longer than you');

// quote = quote.slice(0, quote.indexOf("you")) + "always takes longer than" + quote.slice(quote.indexOf("you")-1);

console.log(quote);